src/views/mainLayout.js

```diff
@@ -14,6 +14,28 @@ const mainLayout = data => {
   const metaDescription = data.meta?.socialDescription ?? getMessage('meta-desc-2')
   const pageUrl = `${AppConstants.SERVER_URL}${data.pathname ?? '/'}`
 
+  const showRecruitmentBanner = () => {
+    // Only show if ENVs are set and user language is set to "en".
+    // Otherwise, return empty string
+    if (
+      AppConstants.RECRUITMENT_BANNER_TEXT &&
+      AppConstants.RECRUITMENT_BANNER_LINK &&
+      getLocale().includes('en')
+    ) {
+      return recruitmentBannerMarkup()
+    }
+    return ''
+  }
+
+  const recruitmentBannerMarkup = () => ` 
+    <div class="recruitment-banner" hidden>
+      <a id="recruitment-banner-link" class="text-link" href="${AppConstants.RECRUITMENT_BANNER_LINK}" target="_blank" rel="noopener noreferrer" data-ga="send-ga-pings" data-event-category="Recruitment" data-event-label="${AppConstants.RECRUITMENT_BANNER_TEXT}">${AppConstants.RECRUITMENT_BANNER_TEXT}</a>
+      <button id="recruitment-banner-dimiss" class="dismiss-btn">
+        <svg aria-label="Close" class="x-close-icon" role="button" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M14.348 14.849a1.2 1.2 0 0 1-1.697 0L10 11.819l-2.651 3.029a1.2 1.2 0 1 1-1.697-1.697l2.758-3.15-2.759-3.152a1.2 1.2 0 1 1 1.697-1.697L10 8.183l2.651-3.031a1.2 1.2 0 1 1 1.697 1.697l-2.758 3.152 2.758 3.15a1.2 1.2 0 0 1 0 1.698z"/></svg>
```

Comment:
nit: `aria-label`'s "Close" text should probably be localized.

---

Which, brings up an interesting dilemma:

```sh
git grep -n "Close" src | cat

src/client/js/components/toastAlert.js:116:  <button aria-label="Close">✕</button>
src/views/mainLayout.js:34:        <svg aria-label="Close" class="x-close-icon" role="button" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M14.348 14.849a1.2 1.2 0 0 1-1.697 0L10 11.819l-2.651 3.029a1.2 1.2 0 1 1-1.697-1.697l2.758-3.15-2.759-3.152a1.2 1.2 0 1 1 1.697-1.697L10 8.183l2.651-3.031a1.2 1.2 0 1 1 1.697 1.697l-2.758 3.152 2.758 3.15a1.2 1.2 0 0 1 0 1.698z"/></svg>
```

The `aria-label="Close"` attribute isn't localized on the `toastAlert`'s close button either, but we're seemingly also using an `×` there vs the inlined SVG button here.

So now I wonder if `×` should be an SVG, or if the SVG should be a `×` for consistency&hellip;

Reply:
Yeah. I didn't localize it, because of the other version.

However - I did some digging: `mobile-close-bento-button-title` in the `bento.ftl` is _"Close menu"_

I'm filed a follow-up issue to make sure all the close/dismiss buttons: 
- Consistently use an SVG
- Have localized attributes 
- Have correct ARIA labels

See issue https://github.com/mozilla/blurts-server/issues/3046